runtime/compiler/p/codegen/J9CodeGenerator.cpp

```diff
@@ -728,9 +728,9 @@ J9::Power::CodeGenerator::insertPrefetchIfNecessary(TR::Node *node, TR::Register
       }
    else if (node->getOpCodeValue() == TR::awrtbari &&
             comp()->getMethodHotness() >= scorching &&
-            TR::Compiler->target.cpu.id() >= TR_PPCp6 &&
-              (TR::Compiler->target.is32Bit() ||
-                (TR::Compiler->target.is64Bit() &&
+            self()->comp()->target().cpu.id() >= TR_PPCp6 &&
+              (self()->comp()->target().is32Bit() ||
+                (self()->comp()->target().is64Bit() &&
                  comp()->useCompressedPointers() &&
```

Comment:
Do these have to use `self()`? The one right below just calls `comp()`

Reply:
🤷‍♂ not sure, in an extensible class I always use `self()` just in case.